include/oboe/FullDuplexStream.h

```diff
@@ -41,7 +41,9 @@ namespace oboe {
  * Caller is responsible for closing both streams.
  *
  * Callers should handle error callbacks with setErrorCallback() for the output stream.
- * When an error callback occurs for the output stream, the caller should restart both streams.
+ * When an error callback occurs for the output stream, Oboe will stopping the output stream.
+ * The caller is responsible for stopping the input stream.
```

Comment:
Caller must stop AND CLOSE the input streams.
Then they can reopen them.

LiveEffect would be a good place to demonstrate this.

Reply:
Done! Thanks for the suggestion!